csharp/ql/test/library-tests/csharp7/LocalTaintFlow.ql

```diff
@@ -2,5 +2,7 @@ import csharp
 import semmle.code.csharp.dataflow.TaintTracking
 
 from DataFlow::Node pred, DataFlow::Node succ
-where TaintTracking::localTaintStep(pred, succ)
+where
+  TaintTracking::localTaintStep(pred, succ) and
+  not pred.asExpr().fromLibrary()
```

Comment:
Why not `pred.asExpr().fromSource()`

Reply:
I tried that, but that would remove SSA definitions, nodes for implicit `this` expressions, and maybe more.

Reply:
👍